tests/test-performance.js

```diff
@@ -0,0 +1,26 @@
+var performance = require("performance");
+
+var total = 0;
+var passed = 0;
+
+function assert(actual, description) {
+    total += 1;
+    var label = "\033[1m\033[31mFAIL\033[0m";
+    if (actual === true) {
+        passed += 1;
+        label = "\033[1m\033[32mPASS\033[0m";
+    }
+    console.log(label + " - " + description);
+}
+
+var before = performance.now();
+//console.log("performance.now() before:", before);
+
+setTimeout(function() {
+    var after = performance.now();
+    //console.log("performance.now() after:", after);
+    var diff = after - before;
+    // Allow for some jitter, especially on Linux
+    assert(diff > 990 && diff < 1010, "performance.now() result over known delay");
```

Comment:
Yeah that really was what I saw pretty consistently, 30 before, 1040 after, on reboot. Now that's **if I enabled the console prints you'd commented out**. Without them it seems to generally pass every time. So maybe the print itself is interfering somehow. I can't say I understand things to that level yet.